content/blogposts/2024/sourcegraph-release-may-2024.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Sourcegraph May 2024 release"
+publishDate: 2024-05-09T10:00-07:00
+description: "The latest Sourcegraph release includes a new point-in-time search filter for Code Search."
+tags: [blog, release]
+slug: "release/May-2024"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+changelogItems:
+  - description: "Improved the performance of language stats insights by 50-70%, significantly reducing pain for customers with large repos. The timeout for language stats insights has also been raised from 3 to 5 minutes. Both are configurable using environment variables on the frontend container."
+    url: https://sourcegraph.com/docs/code_insights/explanations/administration_and_security_of_code_insights#language-stats-performance-configuration
+    category: Code Insights
+  - description: "Improved Zoekt CPU usage and scalability for customers with a large number of repos."
+    url:
+    category: Search
+  - description: "Greatly improved git blame performance on large files, reducing the time it takes for a git blame on a large file to load from several seconds down to less than 500 milliseconds."
+    url:
+    category: Search
+  - description: "Search Jobs now supports diff, commit, and path searches. Before, only file searches were supported."
+    url:
+    category: Source
+ 
+
+
+---
+
```

Comment:
Is there room to add any narrative to this post? Perhaps about how we're making it easier to see into your codebase not just today but at any point in history?

Sourcegraph already gives you viewability into massive codebases across every code host, but now we do it across time, too. It's almost like our search is becoming more 3 dimensional. Thoughts? Too much?

Reply:
Added!